dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -54,8 +97,44 @@ def _CreateHunt(
     """
     runner_args = self.grr_api.types.CreateHuntRunnerArgs()
     runner_args.description = self.reason
+
+    if self.match_mode:
+      if self.match_mode.lower() == 'any':
+        match_mode = runner_args.client_rule_set.MATCH_ANY
+      elif self.match_mode.lower() == 'all':
+        match_mode = runner_args.client_rule_set.MATCH_ALL
+      else:
+        self.ModuleError(f'Unknown match mode {self.match_mode}', critical=True)
+
+      runner_args.client_rule_set.match_mode = match_mode
+
+    if self.client_labels:
+      for client_label in self.client_labels:
+        label_rule = runner_args.client_rule_set.rules.add()
+
+        label_rule.rule_type = label_rule.LABEL
+        label_rule.label.label_names.append(client_label)
+
+    if self.client_operating_systems:
+      client_operating_systems = set(
+          os for os in self.client_operating_systems
+          if os in ('win', 'osx', 'linux'))
```

Comment:
Same comment here - I'd say it's safe to assume that `client_operating_systems` has been validated, so no need for th `if` condition here. In fact, you can remove this block completely and just loop over `self.client_operating_systems` in L 123

Reply:
Removed.